### DISCUSSION TITLE
Uploading photo creates double image

### DISCUSSION THREAD

**Discussion by oblomow** [LycheeOrg/Lychee#2627] [Bugs] [answered]
### Lychee version

v6.0.0

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

Uploading a single photo causes 2 new identical entries in the album. Removing one of the doubles does not work. Setting VUEJS_ENABLED=false resolves this delete problem.

### Steps to reproduce the issue

1 go to an album in the new lychee
2 select the '+' to upload a photo and browse and upload a photo
3 see that uploaded photos is shown twice in the album

### Diagnostics [REQUIRED]

Diagnostics
-----------
Info: Latest version of PHP is 8.3
Info: Found 17 small that could be generated.
     You can use `php artisan lychee:generate_thumbs small 17` to generate them.
Info: Found 1172 small2x that could be generated.
     You can use `php artisan lychee:generate_thumbs small2x 1172` to generate them.
Info: Found 103 medium that could be generated.
     You can use `php artisan lychee:generate_thumbs medium 103` to generate them.
Info: Found 159 medium2x that could be generated.
     You can use `php artisan lychee:generate_thumbs medium2x 159` to generate them.
Info: Found 5 small images without filesizes.
     You can use `php artisan lychee:variant_filesize 5` to compute them.
Info: Have you considered supporting Lychee? :)

System Information
------------------
Lychee Version (release):                  6.0.0
DB Version:                                6.0.0

composer install:                          --no-dev
APP_ENV:                                   local
APP_DEBUG:                                 false
APP_URL:                                   set
APP_DIR:                                   default
LOG_VIEWER_ENABLED:                        true
VUEJS_ENABLED:                             true
PHOTO_PIPES:                               true

System:                                    FreeBSD
PHP Version:                               8.2.24
PHP User agent:                            Lychee/4 (https://lycheeorg.github.io/)
Timezone:                                  Europe/Amsterdam
Max uploaded file size:                    100M
Max post size:                             100M
Chunk size:                                76.80 MB
Max execution time:                        0
MySQL Version:                             8.4.2

exec() Available:                          yes
Imagick Available:                         1
Imagick Enabled:                           1
Imagick Version:                           1809
GD Version:                                2.3.3
Number of foreign key:                     12 found.

Config Information
------------------
version:                                   060000
check_for_updates:                         1
sorting_photos_col:                        taken_at
sorting_photos_order:                      ASC
sorting_albums_col:                        title
sorting_albums_order:                      DESC
imagick:                                   1
skip_duplicates:                           0
small_max_width:                           0
small_max_height:                          360
medium_max_width:                          1920
medium_max_height:                         1080
lang:                                      en
image_overlay_type:                        date
default_license:                           CC-BY-NC-ND-4.0
compression_quality:                       90
grants_full_photo_access:                  1
delete_imported:                           0
mod_frame_enabled:                         1
mod_frame_refresh:                         30
thumb_2x:                                  1
small_2x:                                  1
medium_2x:                                 1
landing_page_enable:                       1
force_migration_in_production:             0
search_public:                             0
hide_version_number:                       0
SL_enable:                                 0
SL_for_admin:                              0
recent_age:                                1
grants_download:                           1
photos_wraparound:                         1
map_display:                               0
zip64:                                     1
map_display_public:                        0
map_provider:                              OpenStreetMap.org
force_32bit_ids:                           0
map_include_subalbums:                     0
update_check_every_days:                   3
has_exiftool:                              1
share_button_visible:                      1
import_via_symlink:                        0
has_ffmpeg:                                0
location_decoding:                         0
location_decoding_timeout:                 30
location_show:                             0
location_show_public:                      0
rss_enable:                                0
rss_recent_days:                           7
rss_max_items:                             100
prefer_available_xmp_metadata:             0
editor_enabled:                            1
lossless_optimization:                     0
swipe_tolerance_x:                         150
swipe_tolerance_y:                         250
local_takestamp_video_formats:             .avi|.mov
log_max_num_line:                          1000
unlock_password_photos_with_url_param:     0
nsfw_visible:                              1
nsfw_blur:                                 0
nsfw_warning:                              0
nsfw_warning_admin:                        0
nsfw_banner_override:                      
map_display_direction:                     1
album_subtitle_type:                       oldstyle
upload_processing_limit:                   4
new_photos_notification:                   1
legacy_id_redirection:                     1
zip_deflate_level:                         6
default_album_protection:                  1
allow_username_change:                     1
album_decoration:                          layers
album_decoration_orientation:              row
auto_fix_orientation:                      1
use_job_queues:                            0
random_album_id:                           starred
use_last_modified_date_when_no_exif_date:  0
layout:                                    justified
date_format_photo_thumb:                   M j, Y, g:i:s A e
date_format_photo_overlay:                 M j, Y, g:i:s A e
date_format_sidebar_uploaded:              M j, Y, g:i:s A e
date_format_sidebar_taken_at:              M j, Y, g:i:s A e
date_format_hero_min_max:                  F Y
date_format_hero_created_at:               M j, Y, g:i:s A T
date_format_album_thumb:                   M Y
upload_chunk_size:                         0
nsfw_banner_blur_backdrop:                 0
search_pagination_limit:                   1000
search_minimum_length_required:            4
photo_layout_justified_row_height:         320
photo_layout_masonry_column_width:         300
photo_layout_grid_column_width:            250
photo_layout_square_column_width:          200
photo_layout_gap:                          12
display_thumb_album_overlay:               always
display_thumb_photo_overlay:               hover
use_album_compact_header:                  0
default_album_thumb_aspect_ratio:          1/1
login_button_position:                     left
back_button_enabled:                       0
back_button_text:                          Return to Home
current_job_processing_visible:            1
SA_random_thumbs:                          0
job_processing_queue_visible:              1
login_required:                            0
enable_unsorted:                           1
enable_starred:                            1
enable_recent:                             1
enable_on_this_day:                        1
login_required_root_only:                  1
dark_mode_enabled:                         1
show_keybinding_help_popup:                0
show_keybinding_help_button:               1
disable_se_call_for_actions:               0
enable_se_preview:                         1
slideshow_timeout:                         5
keep_original_untouched:                   1
hide_nsfw_in_smart_albums_and_search:      1

### Browser & System [REQUIRED]

Macbook 15.1 with firefox 132.0 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Duplicate: https://github.com/LycheeOrg/Lychee/discussions/2611

**Reply by oblomow** [reply]
sorry, I somehow missed the original issue. (2611)

**Reply by ildyria** [reply]
No worries :)
